content/en-US/api/2dcontextlostevent.md

```diff
@@ -0,0 +1,34 @@
+---
+recipe: api-interface
+title: '2dcontextlost'
+mdn_url: /en-US/docs/Web/API/2dcontextlost_event
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.2dcontextlost
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/API](https://developer.mozilla.org/en-US/docs/Web/API).**
+
+## Description
+
+The 2d Rendering context losses may be intended by the user agent (to resolve
+resource contention), or may be forced by external factors (e.g. a graphics
+driver reset). In both scenarios, this event will be triggered after the 
+2dRenderingContext has been lost. 
```

Comment:
This reads a little strangely. I think you should focus first on what CanvasRendering2DContextLost _is_ and what it _does_. Something along the lines of:
  "`contextLost` is a javascript event that is triggered when a 2d context on the page has been "lost". Contexts can be lost when a driver crashes, the application runs out of memory..." etc, etc. 
    Then maybe describe why they're useful?

Reply:
I wonder if i need to add why they are useful. I mean we don't have it for other canvas rendering 2d methods or webgl contest lose event. I feel users will find their way to use this function. If you still believe it's better to have it, I will add it later.